src/Server/RestRouterHandlers/RawstoreTableRestRouterHandler.cpp

```diff
@@ -30,12 +30,11 @@ std::map<String, std::map<String, String> > RawstoreTableRestRouterHandler::crea
 void RawstoreTableRestRouterHandler::buildTablesJSON(Poco::JSON::Object & resp, const CatalogService::TablePtrs & tables) const
 {
     Poco::JSON::Array tables_mapping_json;
-
-    std::vector<String> table_names;
+    std::unordered_set<String> table_names;
 
     for (const auto & table : tables)
     {
-        if (std::find(table_names.begin(), table_names.end(), table->name) != table_names.end())
+        if (table_names.find(table->name) != table_names.end())
```

Comment:
`table_names.contains(...)`

Reply:
Can we calculate `shards` and `replication_factor` for table list ? We can do this in a different PR

Reply:
Yes, I will build a related task